auth-api/src/auth_api/services/org.py

```diff
@@ -272,6 +273,8 @@ def _create_payment_settings(
             payment_account_status = PaymentAccountStatus.PENDING
         else:
             payment_account_status = PaymentAccountStatus.FAILED
+            error_payload = response.json()
```

Comment:
There are more than 200 unit tests failing because they did not throw an exception when they failed, possibly due to a 201 status code. Do we still need the exception here?

Reply:
Probably just need to change the pay spec for testing so it returns the right status code? it's just calling a mock? maybe the spec needs to be updated?